docs/release.md

```diff
@@ -0,0 +1,81 @@
+# Release Process
+
+This document outlines the release process of the load-test tool. It is codified in [a Playbook](https://community.mattermost.com/playbooks/playbooks/95pa6a8fcp8p8xsn4frqejcide) that the team follows monthly in the Community server. If you don't have access to the playbook, please ping the team.
```

Comment:
Should we specify which team to ping or do you think it's sufficiently covered by the owners below?

Reply:
Ah, yes, I'll specify it!

Reply:
https://github.com/mattermost/mattermost-load-test-ng/pull/640/commits/666a4a3d75a1a1b55a6345642adbe4d28892545c